test/blackbox-tests/test-cases/vendor/main.t/run.t

```diff
@@ -15,6 +15,12 @@ When compiling vendored code, all warnings should be disabled
   Entering directory 'warnings'
   There should be no OCaml warning!
 
+When compiling vendored code, all alerts should be disabled as well
```

Comment:
Could you put this into a separate test file or directory? E.g. `vendor/alerts.t`?

Reply:
Sure!